functions/index.js

```diff
@@ -1,12 +1,18 @@
 'use strict';
-
-const firebaseFunctions = require('firebase-functions');
-const firebaseAdmin = require('firebase-admin');
-const gcs = require('@google-cloud/storage')();
-const jwt = require('jsonwebtoken');
-const fs = require('fs');
-
+Object.defineProperty(exports, "__esModule", { value: true });
```

Comment:
What's going on with these imports? At the very least it needs an explanatory comment.